eth/core/state/statedb.go

```diff
@@ -214,6 +218,8 @@ func (sdb *stateDB) Prepare(rules params.Rules, sender, coinbase common.Address,
 			sdb.AddAddress(coinbase)
 		}
 	}
+
+	sdb.TransientJournal.Finalize()
```

Comment:
don't need this here. All of the journals are registered by the controller, which invokes Snapshot, RevertToSnapshot, and Finalize on each of the journals when necessary!